Scheduler/Models/ScheduleRecord.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Scheduler.Enums;
+
+namespace Scheduler.Models
+{
+    public class ScheduleRecord
+    {
```

Comment:
models should never have default values in properties, if you want you can create constructor with initialization